components/x-topic-search/src/AllFollowed.jsx

```diff
@@ -3,32 +3,32 @@ import styles from './TopicSearch.scss';
 import classNames from 'classnames';
 
 // transform like this => topic1, topic2 and topic3
-const arrayToSentence = matchingFollowedTopics => {
-	const topicsLength = matchingFollowedTopics.length;
+const arrayToSentence = followedSuggestions => {
+	const topicsLength = followedSuggestions.length;
 
 	if (topicsLength === 1) {
-		return <b>{ matchingFollowedTopics[0].name }</b>;
+		return <b>{ followedSuggestions[0].prefLabel }</b>;
 	} else {
-		return matchingFollowedTopics
+		return followedSuggestions
 		.map((topic, index) => {
 			if (index + 1 === topicsLength) {
 				// the last topic
-				return  <span key={ index }>and <b>{ topic.name }</b></span>
+				return  <span key={ index }>and <b>{ topic.prefLabel }</b></span>
```

Comment:
Practically, it probably won't make much difference in this case, but it's best to avoid using array indexes for `key`, and instead use something that identifies the element value itself.

```suggestion
				return  <span key={ topic.id }>and <b>{ topic.prefLabel }</b></span>
```

This helps p/react know which DOM elements have been added/removed and can make rendering more efficient.

Reply:
👍